### PR TITLE
Example using Remark plugins in react

### PR DESCRIPTION
**Pull request by icew4ll**
Example using Remark plugins in react to address issue:

https://github.com/brillout/vite-plugin-mdx/issues/30

### CODE REVIEW

**Comment by brillout** [comment]
Thanks for the PR. So, the only problem with #30 was that @krevativ forgot to include the CSS, correct?

**Comment by icew4ll** [comment]
The syntax he posted differs slightly from how I was able to get remark-prism to work.  An example, would need the following to work:

1) `vite.config.js`, needed to wrap the plugin in require

```javascript
  remarkPlugins: [
    require('remark-prism')
  ],
```

2) `index.jsx`, import the css

```javascript
import "prismjs/themes/prism-tomorrow.css"
```

3) Have installed "prismjs" and "remark-prism" with node/yarn.

OP submitted a follow up question which may be worth including regarding line numbering which I suspect can be done with prism plugins but haven't been able to get that working yet.

**Comment by brillout** [comment]
How about:
 - We add `remark-prism` to the existing `examples/react` instead of creating a new example.
 - We improve the readme, e.g. the readme mentions <code>// E.g. \`remark-frontmatter\`</code> which we may want to replace with <code>// E.g. \`require("remark-frontmatter")\`</code> or something better.

**Comment by icew4ll** [comment]
Okay, that makes sense.  How do we proceed should I update my fork with these changes?

**Comment by brillout** [comment]
Yes

**Comment by krevativ** [comment]
Regarding the line numbers, there is an open issue in [rehype-prism](https://github.com/mapbox/rehype-prism/issues/17) too. So this seem to be a common problem that people have with this plugins. I've asked too, just in case one of you knows more. But it's not critical. At least for what I need it. Thanks again for your example @icew4ll That helped me a lot.

**Comment by icew4ll** [comment]
Requested changes have been made

File: examples/preact/package.json [hunk 1]
```diff
@@ -5,11 +5,12 @@
...
     "@mdx-js/mdx": "^1.6.22",
     "@mdx-js/preact": "^1.6.22",
-    "preact": "^10.5.12",
-    "@prefresh/vite": "^2.1.0",
-    "vite": "^2.0.0-beta.65"
+    "@prefresh/vite": "^2.2.2",
+    "preact": "^10.5.13",
+    "remark-prism": "^1.3.6",
+    "vite": "^2.3.4",
+    "vite-plugin-mdx": "file:../../"
```
Comment (brillout) [inline examples/preact/package.json:14]: The `file:../../` breaks the README.md instructions of the example.

Otherwise looks good 👍

**Comment by brillout** [comment]
@icew4ll Have you seen the last review comment?

**Comment by brillout** [comment]
@uhoh-itsmaciek Yea let's move the discussion somewhere else. PR is welcome if changes are needed on vite-plugin-mdx's side (I don't believe so though).

**Pull request closed without merging**